alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/rest/RestClient.java

```diff
@@ -131,85 +109,136 @@ public void setProviderConfiguration(ProviderConfig providerConfiguration) throw
             }
 
             SSLContext sslContext = SSLContexts.createSystemDefault();
-
-            CloseableHttpClient httpClient = HttpClients
+            httpClient = HttpClients
                     .custom()
+                    .setConnectionManager(poolHttpConnManager)
                     .setDefaultRequestConfig(clientConfig)
                     .setSslcontext(sslContext)
                     .build();
-            Unirest.setHttpClient(httpClient);
         } else {
-            CloseableHttpClient httpClient = HttpClients
+            httpClient = HttpClients
                     .custom()
```

Comment:
Could replace HttpClients.custom() by HttpClientBuilder.create().useSystemProperties() to use proxies defined in system properties